molecule.yml

```diff
@@ -50,10 +50,14 @@ ansible:
   host_vars:
     docker:
       docker_use_ipv4_nic_mtu: True
+      # Latest version 17.12.1.ce-1.el7.centos has a bug that prevents
+      # testing on travis: https://github.com/docker/for-linux/issues/219
+      docker_version: 17.09.1.ce-1.el7.centos
     docker-inactive:
       docker_systemd_setup: False
     docker-distro:
       docker_install_upstream: False
+      docker_version: 1.12.6
```

Comment:
I'd think that the point of `docker-distro` would be to *not* need to specify the upstream versions. Perhaps the addition of a variable means the testing matrix needs to be x2 -- with and without version. Unless the upstream/no-version case is just currently broken. Either way, we will need some todo/issue for coming back when issue 219 is fixed.

Reply:
There's a change in the latest distro version (1.13) which conflicts with the custom settings in this role and this was a quick fix. If you've no objection I'd rather make a breaking change so this role only supports upstream.

Reply:
If we're working around a breakage, happy to get this out as a quick minor release with the additional functionality. We can discuss the breaking changes elsewhere. Are those next steps captured?

Reply:
I've added https://github.com/openmicroscopy/ansible-role-docker/issues/5

Reply:
Ta.